cmds/tui.c

```diff
@@ -369,20 +369,22 @@ REPORT_FIELD_TIME(REPORT_F_TOTAL_TIME, total, total.sum, total, "TOTAL TIME");
 REPORT_FIELD_TIME(REPORT_F_TOTAL_TIME_AVG, total-avg, total.avg, total_avg, "TOTAL AVG");
 REPORT_FIELD_TIME(REPORT_F_TOTAL_TIME_MIN, total-min, total.min, total_min, "TOTAL MIN");
 REPORT_FIELD_TIME(REPORT_F_TOTAL_TIME_MAX, total-max, total.max, total_max, "TOTAL MAX");
+REPORT_FIELD_TIME(REPORT_F_TOTAL_TIME_STD, total-std, total.std, total_std, "TOTAL STD");
```

Comment:
I'm curious if the standard deviation still has the same unit with the original values.  I guess it's just a pure number, isn't it?

Reply:
Yes, exactly. It is just a pure number and has the same unit. So what I am trying to do here is to maintain the consistency.

Reply:
Are you sure it has the same unit?  I thought stddev numbers don't have units.

Reply:
Sorry for the confusion here. Indeed, stddev numbers don't have units and is a pure number. The units are generated by the `print_time` function. I'll try to update the commit later today!

Reply:
No, I found stddev would have the same unit of the original data.

https://medium.com/@tvsdhanan009/does-standard-deviation-have-units-40ed74396bba